concent_api/core/transfer_operations.py

```diff
@@ -153,6 +154,50 @@ def create_file_transfer_token_for_golem_client(
     )
 
 
+def create_file_transfer_token_for_verification_use_case(
```

Comment:
Maybe a comment here would be good why using this instead of `create_file_transfer_token_for_golem_client`.

Reply:
done